### PR TITLE
issue#57 Buttons Fixed && issue#58 Routing done

### PR DESCRIPTION
**Pull request by shobhitexe**
## Description

Hey i have fixed the buttons deforming while viewing in mobile + added routing to pages

## Screenshots
Buttons 👇
image

Routing 👇
image
image

## Related Issue

issue#57 and issue#58
 -->
## Checklist

Please review and check off the following items before submitting your PR:

 

- [x ] I have tested my changes to ensure they work as expected
- [x ] I have included documentation for any new features or changes
- [x ] I have formatted my code according to the project's style guidelines
- [x ] I have run the project's test suite and verified that all tests pass
- [x ] I have checked for any spelling or grammatical errors

## Additional Information

It was fun contributing to this project plz accept my pull req

### CODE REVIEW

**Comment by priyankarpal** [comment]
hi, @shobhitexe there is a conflicts error. Please resolve it. 



one suggestion. 
- Add a nav-link "projects"
- remove the back button

**Comment by shobhitexe** [comment]
> hi, @shobhitexe there is a conflicts error. Please resolve it.
> 
> one suggestion.
> 
> * Add a nav-link "projects"
> * remove the back button

Hey the changes has been pushed plz check

**Comment by priyankarpal** [comment]
I checked the deployment preview. It's ok. But also you need to resolve the error

**Comment by shobhitexe** [comment]
> I checked the deployment preview. It's ok. But also you need to resolve the error

wait on it

**Review by priyankarpal — CHANGES_REQUESTED** [review]
Just resolve the error.

File: src/App.jsx [hunk 1]
```diff
@@ -1,79 +1,17 @@
...
-      </section>
-
-      {/* ⚡ Projects Section Ends */}
-
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="projects" element={<Projects />} />
+      </Routes>
       <Footer />
+      <script async defer src="https://buttons.github.io/buttons.js" />
```
Comment (priyankarpal) [inline src/App.jsx:14]: I think it's not needed.

Comment (shobhitexe) [reply src/App.jsx:14]: Its in your code thought the icons comming from it thats why didn't touch it nvm removing it

Comment (priyankarpal) [reply src/App.jsx]: ok

File: src/CSS/index.css [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+}
+
+.scrollbar::-webkit-scrollbar-thumb {
+  background-color: rgb(166 110 252 / var(--tw-bg-opacity));
+  outline: 1px solid slategrey;
+}
+
+.scrollbar::-webkit-scrollbar-thumb:hover {
+  background: #555;
+}
```
Comment (priyankarpal) [inline src/CSS/index.css:28]: Could add it to Tailwind css?

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/CSS/index.css (hunk 2 above)
Comment (priyankarpal) [reply src/CSS/index.css:28]: All CSS

Comment (shobhitexe) [reply src/CSS/index.css:28]: The whole scrollbar design section ir just this part 
 coz as per ik these arnt availaible for Tailwind 
 whenever i work with tailwind i use these only

Comment (priyankarpal) [reply src/CSS/index.css:28]: > The whole scrollbar design section or just this part coz as per ik these arnt availaible for Tailwind whenever i work with tailwind i use these only

Yeah! I checked. Currently, you don't need to do anything about it. It's working properly.
Thanks for your awesome support 
image

**Pull request merged**